doc/how-to/add_service.rst

```diff
@@ -0,0 +1,41 @@
+.. _howto-add-service:
+
+How to add a new service
+========================
+
+If you set up the MicroCloud without MicroOVN or MicroCeph initially, you can add those services with the command :command:`microcloud service add`::
+
+  sudo microcloud service add
+
+If MicroCloud detects a service is installed but not set up, it will ask to configure the service.
+
+#. Select whether you want to set up distributed storage (if adding MicroCeph to the MicroCloud).
+
+   .. note::
+      To set up distributed storage, you need at least three additional disks on at least three different machines.
+      The disks must not contain any partitions.
+
+   If you choose ``yes``, configure the distributed storage:
+
+   1. Select the disks that you want to use for distributed storage.
+
+      You must select at least three disks.
+   #. Select whether you want to wipe any of the disks.
+      Wiping a disk will destroy all data on it.
+
+   #. You can choose to optionally set up a CephFS distributed file system.
+
+#. Select either an IPv4 or IPv6 CIDR subnet for the Ceph internal traffic. You can leave it empty to use the default value, which is the MicroCloud internal network (see :ref:`howto-ceph-networking` for how to configure it).
```

Comment:
Is there a reason why this is not a substep to the first one? The question is asked only if you add MicroCeph, correct?